backend/src/schema/resolvers/users/blockedUsers.spec.js

```diff
@@ -20,7 +20,7 @@ beforeEach(() => {
       return {
         user: authenticatedUser,
         driver,
-        neode: instance,
+        neode,
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-12-06T09:17:11Z" title="Friday, December 6th 2019, 10:17:11 am +01:00">Dec 6, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -20,7 +20,7 @@ beforeEach(() => {
       return {
         user: authenticatedUser,
         driver,
-        neode: instance,
+        neode: neode,
```


```suggestion
        neode,
```
same as :point_up_2: